### PR TITLE
301 - Turn time limit

### PR DESCRIPTION
**Pull request by akvadrako**
For those new to github, look in the **Files Changed** tab above to see the current version of the rule.

The version I first proposed is:

> **301.** When current turn ends, the next turn begins. A turn ends after all the steps are complete (§201) or after 7 full days, at midnight. If the voting period was at least 3 full days, players who haven't yet voted are counted as ABSTAIN. If the voting period was shorter (for example because the proposal was tabled 4 or more days after the start of the turn), abstaintions count as AGAINST.
> 
> A *simple majority* (§203) means a majority of votes, thus an ABSTAIN vote reduces the required number of votes.

Note that because of rule 105, every player is required to vote. If they don't vote while rule 105 is in effect and the time limit is reached, I believe rule 113 comes into affect and the player forfeits. Perhaps later we can fix this issue. Also because of rule 201, not submitting a proposal in time would also constitute forfeiture.

This is just my interpretation, not an official ruling. Also, please try to play each turn within one week.

If accepted this would give me (301 - 291) * 4/4 = 10 points (see rule 202).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,15 @@
-**Points:** all: 0 target: 200
+Player      | order | points
+----------- | ----- | ------
+akvadrako   | Ba    | 10
+RamonRB     | Be    | 0
+rknegjens   | K     | 0
```
Comment (rknegjens) [inline README.md:5]: I propose to change this to `rknegjens   | K     | 100`

**Review by rknegjens — DISMISSED** [review]
I like the idea of setting a time limit, but converting missed votes to abstain seems too dangerous. In a bad week a rule could get through with just one vote. I would prefer an incremental score penalty for failing at ones voting duty.

File: README.md (hunk 1 above)
Comment (akvadrako) [reply README.md:5]: I was thinking it's a good way to continue playing if many people drop out, but I guess it is unsafe.

**Comment by akvadrako** [comment]
Oops, commented on the wrong spot. I was thinking it's a good way to continue playing if many people drop out, but I guess it is unsafe.

**Comment by rknegjens** [comment]
> Oops, commented on the wrong spot. I was thinking it's a good way to continue playing if many people drop out, but I guess it is unsafe.

Will you amend, or close this PR?

**Comment by akvadrako** [comment]
> Will you amend, or close this PR?

I amended it.

**Review by RamonRB — APPROVED** [review]

**Review by roarandrock — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,4 +1,13 @@
...
+roarandrock | W     | 0
+
+## New Rules
+
+**301.** When current turn ends, the next turn begins. A turn ends after all the steps are complete (§201) or after 7 full days, at midnight. Votes that have not yet been cast count as AGAINST.
```
Comment (rknegjens) [inline README.md:10]: I agree with the default AGAINST for absenteeism, but would prefer some point deduction e.g. 1 point for now until we get a feeling for the value of points.

**Review by rknegjens — CHANGES_REQUESTED** [review]

**Comment by akvadrako** [comment]
What does everyone else think? I don’t mind it but it could also be a separate rule or a later amendment.

File: README.md (hunk 2 above)
Comment (rknegjens) [reply README.md:10]: Dev mentioned that he interprets not voting on time to invoke 113, which means forfeit. That means abstain = forfeit = lose, which seems too harsh for forgetting to vote in a given week. Probably simplest if I just vote against this rule.

**Comment by akvadrako** [comment]
Okay, then rule 301 doesn't pass. I got 3/4 votes so that's 8 points :smile: Now it's Ramon's turn. Good luck :four_leaf_clover:

**Pull request closed without merging**